Dockerfile.dapper

```diff
@@ -45,7 +45,7 @@ RUN apt-get update -qq && apt-get install -y --no-install-recommends \
     docker-ce=5:20.10.* \
     && rm -rf /var/lib/apt/lists/*
 ## install golangci
-RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.57.1
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.64.8
```

Comment:
Suggest using the latest version always, as it should be backward compatible. It doesn't support forward compatibility, so this is why sometimes we might encounter an incompatibility issue with golang-ci. 

Can handle this in another PR.